src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Planta.kt

```diff
@@ -15,25 +13,28 @@ class Menta(anioObtencionSemilla: Int, altura: Float) : Planta(anioObtencionSemi
   override fun daSemillas() = this.esFuerte() || altura > 0.4
```

Comment:
Para no repetir la parte de `this.esFuerte()` en todas las subclases, dos opciones:
1. Ponerlo en la superclase y luego hacer `super.daSemillas() || altura > 0.4`.
2. Hacer un template method: un método auxiliar abstracto `condicionEspecie` en `Planta` que luego cada subclase tenga que definir.